src/library/starrating.cpp

```diff
@@ -20,14 +20,26 @@
 #include "library/starrating.h"
 #include "util/math.h"
 
+// Magic number? Explain what this factor affects and how
 const int PaintingScaleFactor = 15;
 
 StarRating::StarRating(int starCount, int maxStarCount)
         : m_myStarCount(starCount),
           m_myMaxStarCount(maxStarCount) {
+    // 1st star cusp at 0° of the unit circle whose center is shifted to adapt the 0,0-based paint area
     m_starPolygon << QPointF(1.0, 0.5);
-    for (int i = 1; i < 5; ++i)
+    for (int i = 1; i < 5; ++i) {
+        // add QPointF 2-5 to polygon point array, equally distributed on a circumference.
+        // To create a star (not a pentagon) we need to connect every second point.
+        //
+        // understand those equations
```

Comment:
..which I did after some trigonometry coaching. I'm still puzzled why the stars are pointing right, though. Seems like `x` and `y` are swapped.. When I use the debug output to draw a chart in libreoffice calc the star points upwards.